com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py

```diff
@@ -720,6 +721,37 @@ def end_parallel(self):
         endP = Stream(self.topology, oport)
         return endP
 
+    def last(self, size=1):
```

Comment:
yeah, though at the moment the Window class does have a default trigger of `count(1)` which also seems somewhat strange. It might be tied too much to SPL atm.